includes/libraries/omise-plugin/helpers/charge.php

```diff
@@ -2,29 +2,6 @@
 if (! class_exists('OmisePluginHelperCharge')) {
     class OmisePluginHelperCharge
     {
-        /**
-         * @param string  $currency
-         * @param integer $amount
-         * @return string
-         */
-        public static function amount($currency, $amount)
-        {
-            switch (strtoupper($currency)) {
-                case 'THB':
-                    // Convert to satang unit
-                    $amount = $amount * 100;
-                    break;
-
-                case 'JPY':
-                    break;
-
-                default:
-                    break;
-            }
-
-            return $amount;
-        }
-
```

Comment:
@jacekstan Nice catch. 
I'm gonna need to keep `OmisePluginHelperCharge::isChargeObject` for a while until I can completely get rid of `includes/libraries/omise-plugin/helpers/charge.php` and `includes/classes/class-omise-charge.php` classes since this is the only way to handle an error object from Omise API. 

But we can remove `Omise_Charge::is_charge_object` right now.

(btw, can you review this https://github.com/omise/omise-magento/blob/master/Model/Api/Charge.php class? I'm thinking to implement `API Model` approach that been using in Omise-Magento to this plugin. Feel free to share if you have any ideas)